panels/updates/meson.build

```diff
@@ -0,0 +1,52 @@
+panels_list += cappletname
+desktop = 'gnome-@0@-panel.desktop'.format(cappletname)
+
+desktop_in = configure_file(
+  input: desktop + '.in.in',
+  output: desktop + '.in',
+  configuration: desktop_conf
```

Comment:
Nitpick: please add trailing commas to every last line here

Reply:
Addressed. It should be addressed when I push the next iteration